src/Serilog.Sinks.Elasticsearch/Sinks/ElasticSearch/ElasticSearchSink.cs

```diff
@@ -261,5 +255,45 @@ private void HandleResponse(IEnumerable<LogEvent> events, DynamicResponse result
                 HandleException(result.OriginalException, events);
             }
         }
+        
+        sealed class ElasticCreateAction
```

Comment:
- [ ] Should we move the DTO:s to a separate file?

Reply:
as they are very specific to this implementation and internal, I would be fine keeping them here.